### PR TITLE
v6 release blog post

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

This patch adds the blog post for the `v6` release.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

Please review!

### CODE REVIEW

File: website/blog/2020-02-22-webdriverio-v6-released.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+![improved flamechart](./assets/flamechart.jpg)
+
+### How to Update?
+
+These improvements come for free and you don't need to do anything to better performance in v6 when upgrading.
+
+## Service Configurations
+
+We are very proud about the amount of different [services](https://www.npmjs.com/search?q=wdio-service) and [reporters](https://www.npmjs.com/search?q=wdio-reporter) that have been build by the community. All these additional plugins require specific configurations in your `wdio.conf.js` and we want to make sure that all these settings are being made standardised structure. Until v5 of WebdriverIO specific options to services and reporters could be defined anywhere in the `wdio.conf.js`, e.g. the Sauce service:
```
Comment (laurentbristiel) [inline website/blog/2020-02-22-webdriverio-v6-released.md]: that have been build by the community => that the community built

File: website/blog/2020-02-22-webdriverio-v6-released.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    - [react$](https://webdriver.io/docs/api/element/react$.html)
+    - [react$$](https://webdriver.io/docs/api/element/react$$.html)
+    - [scrollIntoView](https://webdriver.io/docs/api/element/scrollIntoView.html)
+    - [waitForClickable](https://webdriver.io/docs/api/element/waitForClickable.html)
+    - [waitForDisplayed](https://webdriver.io/docs/api/element/waitForDisplayed.html)
+    - [waitForEnabled](https://webdriver.io/docs/api/element/waitForEnabled.html)
+    - [waitForExist](https://webdriver.io/docs/api/element/waitForExist.html)
+    - [waitUntil](https://webdriver.io/docs/api/element/waitUntil.html)
+
+If you use TypeScript in your project it should automatically tell you to all places that needs to be updated. If you don't use TypeScript which recommend to just search for every command in your code base and modify it accordingly. It should be a pretty mechanical and straight forward task.
```
Comment (laurentbristiel) [inline website/blog/2020-02-22-webdriverio-v6-released.md]: straightforward (one word)

File: website/blog/2020-02-22-webdriverio-v6-released.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+### How to Update?
+
+If you already use an assertion library like [Chai](https://www.chaijs.com/) you can continue doing, especially if you are not interested using `expect-webdriverio`. However you can also start writing new assertions with the new assertion API and maintain two different types of assertion libraries until you decide to get rid of the other.
+
+## Other Changes
+
+Next to all major updates that were described above there are also some minor changes that are worth mentioning:
+
+- __TypeScript Support:__ we improved the typings for WebdriverIO and WebDriver to include better descriptions and more detail
+- __WebDriver Default Path:__ we changeed default WebDriver path to from `/wd/hub` to `/` as most of the browser drivers now default to this, this should have now affect for you - however if you have trouble connecting to a WebDriver endpoint after the upgrade, this could be a reason for that issue
```
Comment (laurentbristiel) [inline website/blog/2020-02-22-webdriverio-v6-released.md]: changeed => changed

**Comment by christian-bromann** [comment]
@klamping @laurentbristiel thanks for the review, fixed all the grammar bugs

**Comment by mgrybyk** [comment]
we have also changed before/after Test/Hook hooks signatures. Should it be mentioned here?

**Comment by christian-bromann** [comment]
> Should it be mentioned here?

We should!

**Comment by christian-bromann** [comment]
@mgrybyk added hook signature changes in the blog post - ready to land?

**Pull request merged**